docs/studios/unity/tutorials/github_actions.md

```diff
@@ -0,0 +1,309 @@
+---
+sidebar_label: 'Testing with GitHub Actions'
+---
+
+# Setting up QA Bots with GitHub Actions
+
+This tutorial will walk you through setting up a QA Bot for your Unity project using GitHub Actions. This tutorial assumes you have
```

Comment:
Agree on the title, I think it'd make sense to update any references to "QA Bot" within the guide to just "bot" since realistically this could be _any_ kind of bot for now (validation, playtest, ones we create in the future)

Reply:
I still see references to "QA Bot". Are we planning on keeping this term?

Reply:
removed most of these except for where I talk specifically about this use cases in the guide!